Pages/BaseTypes.razor

```diff
@@ -0,0 +1,3 @@
+﻿@page "/basetypes"
+
+<h1>Datenbank Typen</h1>
```

Comment:
Was macht das? Was kommt hier hin.
Auserdem: "/basetypes" -> "/basis-typen" oder etwas in der richtung